field_dims:
  x_min: 0
  x_max: 8.21
  y_min: 0
  y_max: 15.95

init_dims:
  x_min: 0
  x_max: 8
  y_min: 2
  y_max: 4

num_particles: 200

noise_stdev:
  position: 0.025
  rotation: 0.001
  # Modifiers for noise stddev when robot is moving or stopped
  # Allows for more accurate localization when stopped by using
  # a lower stddev for noise when the robot is at rest.
  stopped_pos_noise_multiplier: 1.0
  stopped_rot_noise_multiplier: 1.0
  moving_pos_noise_multiplier: 1.0
  moving_rot_noise_multiplier: 1.0

rotation_threshold: 0.25

camera_sigmas: [ 0.500, 0.500 ]

bearing_only: false

beacons:
  - [2.40, 0.0]
  - [6.50, 0.0]
  - [1.71, 15.98]
  - [5.81, 15.98]
